nacc_attribute_deriver/attributes/nacc/modules/uds/apoe_derived.py

```diff
@@ -1,11 +1,11 @@
 from nacc_attribute_deriver.attributes.attribute_collection import AttributeCollection
-from nacc_attribute_deriver.attributes.base.namespace import SubjectDerivedNamespace
+from nacc_attribute_deriver.attributes.base.uds_namespace import UDSNamespace
 from nacc_attribute_deriver.symbol_table import SymbolTable
 
 
 class APOEDerivedAttributeCollection(AttributeCollection):
     def __init__(self, table: SymbolTable):
-        self.__subject_derived = SubjectDerivedNamespace(table)
+        self.__uds = UDSNamespace(table)
```

Comment:
This should be using a subject-level namespace. Maybe need to make one that manages longitudinal/cross-sectional values